bindings-go/go.mod

```diff
@@ -0,0 +1,38 @@
+module github.com/specularL2/specular/bindings-go
+
+go 1.20
+
+replace github.com/ethereum/go-ethereum => ../services/el_clients/go-ethereum
+
+require github.com/ethereum/go-ethereum v0.0.0-00010101000000-000000000000
```

Comment:
Set the version to `v1.13.2`?

Reply:
go mod tidy does that when using replace

Reply:
changed back